pages/payment.ts

```diff
@@ -0,0 +1,77 @@
+import {clickOnElement, fillInInput, waitForVisible} from "../utils.ts";
+import {type Page} from "@playwright/test";
+
+const chooseYourPaymentMethodText = 'payment-plans';
+const quarterlyPlan = 'plan-period-3';
+const annualPlan = 'pplan-period-12';
+const cards = 'payment-method-credit-card';
+const submitPayment = 'submit-payment';
+const cardNumber = '//*[@aria-label="Card number"]';
+const cardExpiryDate = '//*[@aria-label="Expiry date"]';
+const cardCvvCvc = '//*[@aria-label="Security code"]';
+const frameName1 = '//*[@title="Iframe for secured card number"]';
+const frameName2 = '//*[@title="Iframe for secured card expiry date"]';
+const frameName3 = '//*[@title="Iframe for secured card security code"]';
+
+const waitForPaymentPage = async (page: Page) => {
+    await waitForVisible(page, chooseYourPaymentMethodText);
+}
+
+const chooseYourPlan = async(page: Page, planType: string) => {
+    if(planType=='Quarter'){
+        await waitForVisible(page, quarterlyPlan);
+        await clickOnElement(page, quarterlyPlan);
+        await page.waitForTimeout(2500);
+    }if(planType=='Annual'){
+        await waitForVisible(page, annualPlan);
+        await clickOnElement(page, annualPlan);
+        await page.waitForTimeout(2500);
+    }
+}
+
+const chooseYourPaymentMethod = async(page: Page, paymentMethod: string) => {
```

Comment:
It's better to have more specific TS types like:
```suggestion
const chooseYourPaymentMethod = async(page: Page, paymentMethod: 'Cards' | 'iDeal') => {
```

This is just an example and you should use the valid values.